README.md

```
@@ -54,15 +54,33 @@ python --version
 
-### Install the toolkit
+### Install channels
 
-Open a terminal and run:
+There are two channels: **stable** (recommended) and **nightly** (latest development build).
+
+#### Stable
+
+The latest reviewed, tested release:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)